codespell_lib/data/dictionary.txt

```diff
@@ -41251,44 +41277,79 @@ suggesttion->suggestion
 suggesttions->suggestions
 sugget->suggest
 suggeted->suggested
+suggeting->suggesting
+suggetion->suggestion
+suggetions->suggestions
 suggets->suggest, suggests,
 suggetsed->suggested
 suggetsing->suggesting
 suggetsion->suggestion
+suggetsions->suggestions
+suggetss->suggests
 sugggest->suggest
 sugggested->suggested
 sugggesting->suggesting
 sugggestion->suggestion
 sugggestions->suggestions
+sugggests->suggests
 sugguest->suggest
 sugguested->suggested
 sugguesting->suggesting
 sugguestion->suggestion
 sugguestions->suggestions
+sugguests->suggests
 suh->such
 suiete->suite
 suiteable->suitable
 sumamry->summary
 sumarize->summarize
 sumary->summary
+sumbission->submission
+sumbissions->submissions
+sumbit->submit
+sumbits->submits
 sumbitted->submitted
+sumbitting->submitting
 sumed-up->summed-up
 summar->summary, summer,
 summarizen->summarize
 summay->summary
+summeries->summaries
+summerisation->summarisation
+summerise->summarise
 summerised->summarised
+summerises->summarises
+summerising->summarising
+summerization->summarization
+summerize->summarize
 summerized->summarized
+summerizes->summarizes
+summerizing->summarizing
 summersalt->somersault
 summmaries->summaries
 summmarisation->summarisation
+summmarise->summarise
 summmarised->summarised
+summmarises->summarises
+summmarising->summarising
 summmarization->summarization
+summmarize->summarize
 summmarized->summarized
+summmarizes->summarizes
+summmarizing->summarizing
 summmary->summary
+summuries->summaries
+summurisation->summarisation
+summurise->summarise
+summurised->summarised
+summurises->summarises
+summurising->summarising
+summurization->summarization
 summurize->summarize
 summurized->summarized
 summurizes->summarizes
 summurizing->summarizing
+summury->summary
```

Comment:
Perhaps `summery` too? It's uncommon though.

Reply:
Added to dictionary_rare.

Reply:
Oh, I meant:
```
summury->summary, summery,
```